src/views/partners.hbs

```diff
@@ -4,20 +4,10 @@
 {{> main/title title="Partner"}}
 <div class="container">
   <div class="d-flex flex-wrap">
-    <h2 class="mb-3">Wir bedanken uns herzlich bei unseren bisherigen Förderpartnern: </h2>
-    <ul id="partner-names" class="two-columns p-0 d-none d-sm-block" style="list-style-type:none;">
-      {{#each partners}}
-        <li class="mb-1"> {{name}} </li>
-      {{/each}}
-    </ul>
-    <div id="partner-names" class="d-sm-none" style="list-style-type:none;">
-      {{#each partners}}
-        <div class="mb-2"> {{name}} </div>
-      {{/each}}
-    </div>
+    <h2 class="mb-4">Wir bedanken uns herzlich bei unseren bisherigen Förderpartnern: </h2>
```

Comment:
Let's remove `bisherigen`, it is kind of odd.

Reply:
Done